busybox/test_busybox.c

```diff
@@ -0,0 +1,44 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+#include <dirent.h>
+#include <errno.h>
+
+int main(int argc, char **argv)
+{
+	int ret;
+	char cmdargs[20];
+	char cmd[140] = "cd /usr/test/busybox/testsuite/ && export PATH=/bin:/sbin:/usr/bin:/usr/sbin && export bindir=/bin && ./runtest";
+	DIR *dir = opendir("/usr/test/busybox/testsuite");
+
+	if (dir) {
+		closedir(dir);
+	}
+	else if (ENOENT == errno) {
+		fprintf(stderr, "There is no busybox test suite to run, build project with \"LONG_TEST = 'y'\"\n");
+		return 1;
+	}
+	else {
+		fprintf(stderr, "There is problem with opening existing /bin/testsuite directory: %s\n", strerror(errno));
+		return 1;
+	}
+
+	if (argc == 2)
+		sprintf(cmdargs, " -v %s", argv[1]);
```

Comment:
nit: maybe is worth to check if `argv[1]` fits into `cmdargs` buffer to avoid bugs in the future development